packages/devops/configs/servers.conf

```diff
@@ -644,38 +584,14 @@ server {
 
   listen       80;
   listen       [::]:80;
-  server_name  tupaia.org www.tupaia.org ~^(?<subdomain>((?!^www).)+)\.tupaia\.org$;
-  root         /home/ubuntu/tupaia/packages/web-frontend/build/desktop;
-  set $mobile_rewrite do_not_perform;
-  set $hostred '';
+  server_name  tupaia.org www.tupaia.org ~^(?<subdomain>((?!^www).)+)\.tupaia\.org$ ~^.*mobile\.tupaia\.org$;
```

Comment:
Adding `mobile.tupaia.org` urls here so that any existing mobile links will still function correctly

Reply:
Hmm... So this configuration is just making `mobile.tupaia.org` another valid url. We _might_ want to configure this so that `mobile.tupaia.org` redirects to regular `tupaia.org`, but honestly I don't know enough about nginx to make that happen. Maybe we can keep that ticket around for that task